packages/zoe/src/contracts/mintPayments.js

```diff
@@ -5,68 +5,73 @@ import harden from '@agoric/harden';
 import produceIssuer from '@agoric/ertp';
 import { makeZoeHelpers } from '../contractSupport';
 
-/*
-This is the simplest contract to mint payments and send them to users
-who request them. No offer safety is being enforced here.
-*/
+/**
+ * This is the simplest contract to create a new issuer and mint payments from
```

Comment:
I think we should mention something about this being more of an example and not necessary meant to be reused as is. Maybe it needs to be changed to be reusable

Reply:
I changed the description to say that it's intended more as an example than as a contract that would be reused.